### ISSUE TITLE
Teachers and students stuck in rooms on last level

### ISSUE THREAD

**Issue by cgeorg3** [JulienNigon/SpellcasterUniversity-Bug-Report#760]
Everytime I get to the last level (Turtle) the majority of the students and teachers become trapped in different rooms. The amount of mana produced is significantly less when this occurs making it impossible to complete the challenges.

**Comment by Neelan22** [comment]
Yes, you're not the only one, we're gonna have a look at this. It's restrictive in game... :/
 #747
